internal/auth/oidc/request/request.pb.go

```diff
@@ -1,7 +1,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
 // 	protoc-gen-go v1.27.1
-// 	protoc        v3.17.3
+// 	protoc        (unknown)
```

Comment:
This is the only change to all of the generated files, and it's introduced because `buf` does not send any version information to generators.